mttools/geometry_tools/__init__.py

```diff
@@ -5,7 +5,15 @@
 from math import sqrt, pi
 from typing import List
 
-from mttools.utils.types import Number
+from mttools.utils.types import Number, RealNumber
+from mttools.utils.exceptions import DimensionError
+from .PropertiesOfBodies import (
```

Comment:
I would like users to import the functions from the `PropertiesOfBodies` file. This adds more information when importing and follows the same practice and the rest of the `foo_tools` sections.